src/FEValues/GeometryMapping.jl

```diff
@@ -149,6 +166,25 @@ end
 calculate_detJ(J::Tensor{2}) = det(J)
 calculate_detJ(J::SMatrix) = embedding_det(J)
 
+function calculate_mapping_and_spatial_coordinate(gip::VectorizedInterpolation, ξ::Vec, cell_coordinates::AbstractVector{<:Vec})
+    return calculate_mapping_and_spatial_coordinate(gip.ip, ξ, cell_coordinates)
+end
+
+function calculate_mapping_and_spatial_coordinate(gip::ScalarInterpolation, ξ::Vec{rdim,Tξ}, cell_coordinates::AbstractVector{<:Vec{sdim, Tx}}) where {Tξ, Tx, rdim, sdim}
```

Comment:
Since there is no information about the function interpolations to be mapped, I don't see why we should return a `MappingValues` here. 
```suggestion
function calculate_jacobian_and_spatial_coordinate(gip::ScalarInterpolation, ξ::Vec{rdim,Tξ}, cell_coordinates::AbstractVector{<:Vec{sdim, Tx}}) where {Tξ, Tx, rdim, sdim}
```